core/relay/config/Corda_Relay2.toml

```diff
@@ -1,40 +1,52 @@
-name = "Corda_Relay"
-port="9080"
+name = "Corda_Relay2"
+port="9082"
```

Comment:
I think this change should be reverted. `Corda_Relay2` needs to know how to reach `Corda_Relay`, not itself.

Reply:
This is defining the name and port for `Corda_Relay2`. Not implying how to reach any relay.

Reply:
But why does `Corda_Relay2` need its own definition rather than the other Corda network's definition?

Reply:
I understand now. I assumed, by mistake, that the above configuration lay in the relay section.